### DISCUSSION TITLE
Using BGPq3 with peering-manager

### DISCUSSION THREAD

**Discussion by moitrierk** [peering-manager/peering-manager#631] [Q&A] [answered]
Dear Community,

I installed peering-manager with Docker.
Everything works fine !

I would like to be able to use BGPq3 with peering-manager, but I don't understand well how it's integrate in Peering-Manager.
I found an entry in the file environment named BGPQ3_PATH, BGPQ3_HOST, BGPQ3_SOURCES.
I saw these variables in the documentation of peering-manager, but I don't really now how I could use directly from the webpage of Peering-Manager or via jinga template.

If someone could share his experience on that topic.

Thanks a lot in advance,

Best regards,
Kevin

**Comment by wtremmel** [comment, marked as answer]
See here: https://peering-manager.readthedocs.io/en/stable/templating/filters/#prefix_list

BGPq3 is used to build prefix lists

**Reply by moitrierk** [reply]
Thanks a lot !
